### PR TITLE
[AIFBR-10395] Add Orchestrator Script for 21.4 removing the ports dependency

### PR DESCRIPTION
**Pull request by Mdirfan6**
Changes have been tested by providing the storage and aif url

### CODE REVIEW

File: orchestrator/orchestratorAutomationAIC21-10.ps1 [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+.EXAMPLE
+    If aifabric is available at ww.xx.yy.zz:port, command to run would be
+    .\orchestratorAutomation.ps1 -aifUrl ww.xx.yy.zz:port
+
+    If ai-app is accessed via domain instead of IP:PORT combo, then enable domainBasedAccess to true
+    .\orchestratorAutomation.ps1 -aifUrl "aif-sahil-aks.westeurope.cloudapp.azure.com"
+
+    If Orchestrator Installation Path has to be specified,
+    .\orchestratorAutomation.ps1 -aifUrl ww.xx.yy.zz -config "C:\Program Files (x86)\UiPath\Orchestrator"
+
```
Comment (mohitseth-nitrr) [inline orchestrator/orchestratorAutomationAIC21-10.ps1:23]: Fix the example in Notes.

**Review by mohitseth-nitrr — CHANGES_REQUESTED** [review]
Changes needed

File: orchestrator/orchestratorAutomationAIC21-10.ps1 (hunk 1 above)
Comment (mohitseth-nitrr) [reply orchestrator/orchestratorAutomationAIC21-10.ps1]: @Mdirfan6 I can still see examples where aifUrl is provided but storageUrl is not. Both are mandatory going forward.

**Review by mohitseth-nitrr — CHANGES_REQUESTED** [review]
More changes needed

File: orchestrator/orchestratorAutomationAIC21-10.ps1 (hunk 1 above)
Comment (Mdirfan6) [reply orchestrator/orchestratorAutomationAIC21-10.ps1]: Updated changes as mentioned.

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Review by SahilChaudhary25 — APPROVED** [review]
LGTM

**Pull request merged**